vendor/golang.org/x/sync/semaphore/semaphore.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
nit: We prefer to have the vendor/autogenerated files in a separate commit. It should be ok as the new files are not too many.